test/desugar/try.txt

```diff
@@ -0,0 +1,29 @@
+;;; TOOL: wast-desugar
```

Comment:
I mainly did it to show that the wat-writer is working. I didn't even notice it had a folding version. I will add this as a separate PR.